ReportUtilityBills/measure.rb

```diff
@@ -311,9 +311,9 @@ def get_utility_rates(hpxml_path, fuels, utility_rates, bill_scenario, pv_system
           tariff = tariff[:items][0]
           fields = tariff.keys
 
+          rate.fixedmonthlycharge = 0.0
           if fields.include?(:fixedchargeunits)
             if tariff[:fixedchargeunits] == '$/month'
-              rate.fixedmonthlycharge = 0.0 if fields.include?(:fixedchargefirstmeter) || fields.include?(:fixedchargeeaaddl)
```

Comment:
Bugfix to ensure that `rate.fixedmonthlycharge` is never nil, so that the line `rate.fixedmonthlycharge += monthly_fee` further down does not generate an error.